src/main/java/org/jabref/gui/maintable/MainTable.css

```diff
@@ -2,48 +2,51 @@
     -fx-border-width: 0;
     -fx-padding: 0;
     -fx-border-insets: 0;
-    -fx-background-color: -fx-light-background;
     -fx-table-cell-border-color: transparent;
+    /*-fx-background-color: -jr-foreground;*/
 }
 
 .table-view .column-header-background {
-    -fx-background-color: -fx-light-background;
+    -fx-background-color: -fx-control-inner-background;
     -fx-border-width: 0 0 1 0;
-    -fx-border-color: darkgrey;
+    -fx-border-color: -fx-background;
 }
 
 .table-view .column-header-background:hover {
-    -fx-background-color: darkgray;
+    -fx-background-color: derive(-fx-control-inner-background, -10%);
 }
 
+
 .table-view .column-header,
 .table-view .filler {
-    -fx-background-color: transparent, -fx-light-background;
+    -fx-background-color: transparent, -fx-control-inner-background;
     -fx-background-insets: 0, 0 0.02em 0 0.02em;
     -fx-font-weight: bold;
     -fx-size: 3em;
+    -fx-border-width: 0 0 1 0;
+    /*-fx-border-color: -jr-separator;*/
 }
 
 .table-view .column-header > .label {
     -fx-padding: 0 1em 0 1em;
     -fx-alignment: center-left;
-    -fx-text-fill: -fx-unimportant;
+    -fx-text-fill: -jr-theme;
 }
 
 .table-view .column-header .glyph-icon {
     -fx-alignment: baseline-center;
-    -fx-text-fill: -fx-unimportant;
-    -fx-fill: -fx-unimportant;
+    -fx-text-fill: -jr-icon;
+    -fx-fill: -jr-icon;
 }
 
 .table-cell {
     -fx-padding: 0.5em 1em 0.5em 1em;
     -fx-cell-size: 4.0em;
 }
 
-.table-row-cell:odd {
-    -fx-background: -fx-control-inner-background;
-}
+/*.table-row-cell:odd {*/
```

Comment:
Please reactivate this, because otherwise the rows are styled in a (ugly) strip-pattern.

Reply:
I disagree. This is a usabiltily enhancement to difference between odd and even rows. Makes it much easier to see which entry is in which row.  So called "Zebra Striping"
http://uxmovement.com/content/9-design-techniques-for-user-friendly-tables/
https://alistapart.com/article/zebrastripingmoredataforthecase

Reply:
See also here for some general ideas about desinging data tables:
https://uxdesign.cc/design-better-data-tables-4ecc99d23356

Reply:
Well, I don't want to start a rant about zebras (they are quite beautiful animals), but anyway here is the rant:
First of all, there are indeed a few studies (like [this one](http://alistapart.com/article/zebrastripingmoredataforthecase)) which show that zebra strips help users in some cases. One should note that the improvement in performance is relatively small (a few 1-3%) and only when the user is required to move vertically in a row to look for data. It is also still controversial whether the better performance comes from a superior design or just because testers where used to the striped style. On the other hand, zebra strips actually lead to worse performance when the task is to quickly identify a desired row containing given data because the eye takes these lines as fix points instead of scan quickly through the dataset. 

So zebra strips may have a place in bigger datasheets where it is required to jump in a specific row and the data spans multiple columns. We have, on the other hand, a very small dataset with like 20 entries on display and 5 or so columns. Moreover, and this is probably the biggest point, a row is highlighted when selected or hovered and thus you get all the benefits of the zebra style without the clean look of the plain table. 

Sorry for the rant but this is one of the things where people just follow the general advice in the form "zebra strips help to navigate in data table" without asking what are the boundary conditions for the performance increase and if they apply in the present case. This leads to a lot of documents /blog posts citing this as a always-good style to present data in a table format.

Reply:
lets ask the other @JabRef/developers  on their opinion on this. I currently find it very hard to identify an entry in the list.  Especially, if you have maybe some entries in which not all required fields are visible. Compare this to the screenshot in the first post. The version from @halirutan  looks much better with the stripes. 
This is also the case when you have multiple files in the list view.

![grafik](https://user-images.githubusercontent.com/320228/37843810-450c6c90-2ec6-11e8-98d4-414eb3408fc1.png)

Reply:
I give my support for using zebra stripping.